draft-ietf-quic-manageability.md

```diff
@@ -156,7 +151,8 @@ The following information is exposed in QUIC packet headers:
 
 - "fixed bit": the second most significant bit of the first octet most QUIC
   packets of the current version is currently set to 1, for demultiplexing
-  with other UDP-encapsulated protocols.
+  with other UDP-encapsulated protocols. Endpoints that do not need this
+  demultiplexing function might negotiate the use of arbitrary values.
```

Comment:
Actually this new sentence does not seem right. The transport draft says:

```
   Fixed Bit:  The next bit (0x40) of byte 0 is set to 1.  Packets
      containing a zero value for this bit are not valid packets in this
      version and MUST be discarded.
```

Reply:
This is another warning about the grease, of course. I think it makes sense to include, but we have the other warning about grease in Section 3.1. I'll delete the sentence to move things along, because it's not a big deal.

Reply:
section 3.1? I thought you actually can grease the fixed bit in QUICv1...?

Reply:
Okay, actually I think it would be important here to note that this bit can/should not be used to identify QUIC (rather than talking specifically about greasing). How about saying something like: "Even though this bit is defined as fixed in the QUICv1 base spec, this bit is not invariant between versions or could even still be changed in an extension, therefore it can not be used an identifier for QUIC."